tests/matmul_checks/check_matsets_ordering.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
```

Comment:
this file could go in `devel/` rather than here

Reply:
where is the devel/ folder located?  I took this file out for now, it was just a quick check I did to make sure things were ordered properly so I don't think it's needed as part of the repo.  Didn't mean to include it actually.